src/Dapr.Client/DaprClient.cs

```diff
@@ -782,6 +783,20 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             IReadOnlyDictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Get a list of configuration items based on keys from the given statestore. 
+        /// </summary>
+        /// <param name="storeName">The name of the statestore to be queried.</param>
+        /// <param name="keys">An optional list of keys to query for. If provided, the result will only contain those keys. An empty list indicates all keys should be fetched.</param>
+        /// <param name="metadata">Optional metadata that will be sent to the statestore being queried.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns></returns>
+        public abstract Task<IReadOnlyList<ConfigurationItem>> GetConfiguration(
```

Comment:
let's figure out how to communciate to users taht this is experimental and may change in the future.

Reply:
Using `Obsolete` as we discussed.